app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -670,15 +670,16 @@ def build_toolbar_hide_button(id)
       when "action_delete"
         return true unless role_allows(:feature => "action_delete")
       end
-    when "MiqAeClass", "MiqAeField", "MiqAeInstance", "MiqAeMethod", "MiqAeNamespace"
+    when "MiqAeClass", "MiqAeDomain", "MiqAeField", "MiqAeInstance", "MiqAeMethod", "MiqAeNamespace"
       return false if MIQ_AE_COPY_ACTIONS.include?(id) && MiqAeDomain.any_unlocked?
+      editable_domain = User.current_tenant.editable_domains.include?(@record)
```

Comment:
Notice, we do this user -> current tenant -> editable domains to see if it includes the domain instead of asking the domain itself, why?